web/settings/modulconfiglp.php

```diff
@@ -995,6 +996,64 @@ function getCookie(cname) {
 
 								</div>
 							</div>
+							<div id="socvag" class="hide">
+								<div class="form-group">
+									<div class="form-row mb-1">
+										<label for="soc_vag_type" class="col-md-4 col-form-label">Fahrzeugtyp</label>
+										<div class="col">
+											<input class="form-control" type="text" name="soc_vag_type" id="soc_vag_type" value="<?php echo $soc_vag_typeold ?>">
```

Comment:
Da es nur drei gültige Optionen gibt, sollte besser ein Optionsfeld mit entsprechenden Select-Elementen genutzt werden. Somit werden Tippfehler gar nicht erst ermöglicht.

Reply:
Am Ende ist es ja das EVCC Modul - durch das Typ-Feld können am Ende eine ganze Reihe von Fahrzeugen -auch abseits VAG - angewählt werden. Aber es steht natürlich im Konflikt mit dem Ansatz, ein SoCModul pro Fahrzeug-/Herstellergruppe zu haben. Dem gewählten Namen VAG treu bleibend, würde ich dann noch Skoda und Porsche(?) hinzufügen als Option, oder was meinst Du? Zudem: In welcher Datei kann ich mir das Erstellen von Optionsfeldern am Besten abschauen?

Reply:
Die benötigten Parameter unterscheiden sich jedoch bei den Fahrzeugen. Daher sollten wir es hier auf VAG beschränken.

Reply:
auf Optionsfelder umgestellt